README.md

```diff
@@ -43,6 +43,7 @@ Currently this library supports synchronising
  - [Categories](/docs/usage/CATEGORY_SYNC.md)
  - [Products](/docs/usage/PRODUCT_SYNC.md) (_Beta_: Not recommended for production use yet.)
  - [InventoryEntries](/docs/usage/INVENTORY_SYNC.md) (_Beta_: Not recommended for production use yet.)
+ - [ProductTypes](/docs/usage/PRODUCT_TYPE_SYNC.md) (_Beta_: Not recommended for production use yet.)
```

Comment:
we can keep it in beta or remove it (the ball is in your court xD). The `Nested`type is in Beta in commercetools, so keep the product type sync in Beta just for that reason maybe is not enough.

I put it in Beta because even after already tested it and so on... I thought it was wise to set it as Beta just for some time until we have confidence that it actually works as the customers expect.

WDYT?

Reply:
> quote 

I put it in Beta because even after already tested it and so on... I thought it was wise to set it as Beta just for some time until we have confidence that it actually works as the customers expect.